convert2rhel/subscription.py

```diff
@@ -1057,7 +1057,7 @@ def update_rhsm_custom_facts():
     the conversion with the candlepin server, thus, propagating the
     "breadcrumbs" from convert2rhel as RHSM facts.
     """
-    if not tool_opts.no_rhsm:
+    if not tool_opts.no_rhsm and not "CONVERT2RHEL_DISABLE_TELEMETRY" in os.environ:
```

Comment:
I want to suggest here a small change that would make a bit more sense and make our lives easier, regading unit testing: 

```suggestion
    if "CONVERT2RHEL_DISABLE_TELEMETRY" in os.environ:
        loggerinst.info("Telemetry disabled, skipping RHSM facts upload.")
        return 

    if not tool_opts.no_rhsm:
```